src/olympia/abuse/cinder.py

```diff
@@ -210,12 +210,13 @@ def post_report(self, *, job):
         a keyword argument."""
         pass
 
-    def workflow_recreate(self, *, job):
+    def workflow_recreate(self, *, notes, job=None):
         """Recreate a job in a queue."""
         raise NotImplementedError
 
-    def workflow_move(self, *, job):
-        """Move job to a different queue."""
+    def post_queue_move(self, *, job):
```

Comment:
It's because it doesn't actually move the job - that API functionality is still under development by Cinder - and the naming implies it did. The function is for what should happen after the move on our side.

Reply:
Okay. 👍 might be worth a docstring to clarify this is a "reactive" action.

Reply:
Nope, didn't see it was already there since it is below the comment card.